.github/actions/markPullRequestsAsDeployed/action.yml

```
@@ -0,0 +1,18 @@
+name: 'Mark Pull Requests as Deployed'
+description: 'Mark pull requests as deployed on production or staging'
+inputs:
+  PR_LIST:
+    description: 'Array or pull request numbers'
+    required: true
+    default: '[]'
+  IS_PRODUCTION_DEPLOY:
+    description: 'Check if deploying to production'
+    required: true
+    default: 'false'
+  GITHUB_TOKEN:
+    description: 'GitHub Token'
+    required: true
+    default: "${{ github.token }}"
+runs:
+  using: 'node12'
+  main: 'dist/index.js'
```

This seems wrong, I think it needs to be:

```suggestion
  main: './index.js'
```